editNickname.php

```diff
@@ -0,0 +1,22 @@
+<?php
+require_once "functions/allFunctions.php";
+
+if (!isset($_POST['id'])){
+    header("location: index.php");
+    exit();
+}
+if (!is_numeric($_POST['id'])){
+    header("location: index.php");
+    exit();
```

Comment:
Maybe you could move both of these if statements into one with an or operator (||)

Reply:
Combined them into an AND statement as this logic table is the the same as ! OR !